core/errors.ml

```diff
@@ -183,6 +184,7 @@ let format_exception =
      pos_prefix (Printf.sprintf "Error: Cannot load plugin '%s' (link error: %s)\n" file (Dynlink.error_message err))
   | ClientCallOutsideWebMode fn ->
      pos_prefix (Printf.sprintf "Error: Cannot call client side function '%s' outside of web mode\n" fn)
+  | MissingBuiltinType alias -> Printf.sprintf "Error: Missing builtin type with alias '%s'. Is it defined in the prelude?" alias
```

Comment:
Is this some expectation mismatch error? Do note that in the (near) future most/all type aliases will be declared and implemented in a Links source file like `prelude.links`.

Reply:
This is the error thrown when the lens code tries to find the `Serial` type (which is already defined in the prelude). See https://github.com/rudihorn/links/blob/65ed306e12cfeee43384a8862deaccd038ca9f02/core/lens_type_conv.ml#L24 and https://github.com/rudihorn/links/blob/65ed306e12cfeee43384a8862deaccd038ca9f02/core/lens_type_conv.ml#L15.

Reply:
Thank you for the clarification.